workers/unity/Packages/com.improbable.gdk.buildsystem/Configuration/BuildConfig.cs

```diff
@@ -54,8 +54,8 @@ private void ResetToDefault()
                             WorkerBuildData.GetCurrentBuildTargetConfig()),
                     CloudBuildConfig = new BuildEnvironmentConfig(
                         WorkerBuildData.AllBuildTargets,
-                        new BuildTargetConfig(BuildTarget.StandaloneOSX, BuildOptions.Development, true),
-                        new BuildTargetConfig(BuildTarget.StandaloneWindows64, BuildOptions.Development, true))
+                        new BuildTargetConfig(BuildTarget.StandaloneOSX, BuildOptions.Development, true, false),
```

Comment:
We're entering the dreaded "bool bool bool" API territory here. At the very least let's use named parameters.

Reply:
This could easily be a tri-state enum: 

```
Disabled
Enabled
Required
```

Reply:
Discussed offline with multiple people - will leave as bools, but will add named parameters